AppCenterDataStorage/AppCenterDataStorageTests/NSObject+MSTestFixture.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NSObject (MSTestFixture)
+
+/**
+ * Return a JSON fixture in a Fixtures folder (relative to the current class).
+ *
+ * @param fixture The name of the fixture to retrieve.
+ * @return The fixture's data.
+ */
+- (NSData *)getJsonFixture:(NSString *)fixture;
```

Comment:
AHh I missed this one, we don't mention `get` when the return type is what we get in objc. But it's nit because in tests so won't block. Whoever find it next fix it.